packages/@o3r/core/src/core/interfaces/configuration.ts

```diff
@@ -13,32 +13,13 @@ export interface NestedConfiguration {
   [key: string]: ConfigurationValueType;
 }
 
-/**
- * @Deprecated will be removed in v10. Please use Configuration instead
- * Generic configuration
- */
-export interface LegacyConfiguration {
-  [key: string]: ConfigurationValueType | Partial<LegacyConfiguration> | (ConfigurationValueType | Partial<LegacyConfiguration>)[] | undefined;
-}
-
 /**
  * Interface of configuration that is supported by the cms
  */
-export interface StrictConfiguration {
+export interface Configuration {
   [key: string]: ConfigurationValueType | (string | NestedConfiguration)[];
 }
 
-/**
- * Configuration mode
- */
-export type ConfigurationMode = 'legacy' | 'strict';
-
-/**
- * Interface of configuration that is supported by the cms
- * NOTE: specify "legacy" as parameter type to disable strict mode and allow values not supported by the CMS. This option is strongly not recommended and deprecated.
- */
-export type Configuration<T extends ConfigurationMode = 'strict'> = T extends 'strict' ? StrictConfiguration : LegacyConfiguration;
```

Comment:
Hmmm I wonder if it would not be safer to keep it with `export type Configuration<T extends 'strict' = 'strict'> = T extends 'strict' ? StrictConfiguration : never;` because this parameter has not be officially deprecated :S.
Or provide an `ngUpdate` to find/replace `Configuration<'strict'>` to `Configuration`

Reply:
ngUpdate provided